### PR TITLE
[#48242] Convert `members_form.js` to Stimulus

### PR DESCRIPTION
**Pull request by aaron-contreras**
Converts the members form dynamic script into Stimulus.

See https://community.openproject.org/work_packages/48242

**NOTE**: Focusing on the `ng-select` element for the autocompleter poised to be tricky. It seems the functionality to autofocus for accesibility-purposes on the autocompleter when the Member form is opened is not currently working on dev (and hasn't been for a while). I'd like some help figuring out (if it's out-of-scope for this, perhaps on a separate PR/Work Package).

### CODE REVIEW

**Review by oliverguenther — CHANGES_REQUESTED** [review]
- [ ] When editing a member, clicking "cancel" does not close the form

**Comment by oliverguenther** [comment]
I pushed a commit to autofocus ng-select, which can be done simply focusing the input field

File: frontend/src/stimulus/controllers/dynamic/members-form.controller.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    this.addMemberFormTarget.style.display = 'block';
+    this.hideFilter();
+    this.filterMemberButtonTarget.classList.remove('-active');
+    window.OpenProject.guardedLocalStorage('showFilter', 'false');
+    this.addMemberButtonTarget.setAttribute('disabled', 'true');
+
+    const select = this.addMemberFormTarget.querySelector<HTMLInputElement>('.ng-input input');
+    select?.focus();
+
+    this.memberUserIdsTarget.addEventListener('mouseover', () => {
```
Comment (oliverguenther) [inline frontend/src/stimulus/controllers/dynamic/members-form.controller.ts]: This event listener never gets removed, prefer to use the `data-action="mouseover->members-form#methodCall"`

Comment (aaron-contreras) [reply frontend/src/stimulus/controllers/dynamic/members-form.controller.ts]: Thank you, still getting used to it :)

**Comment by oliverguenther** [comment]
> * [ ]  When editing a member, clicking "cancel" does not close the form

FYI @aaron-contreras , this is still happening

**Review by oliverguenther — APPROVED** [review]

**Pull request merged**